README.md

```diff
@@ -17,6 +17,8 @@ MCT is developed by researchers and engineers working at Sony Semiconductor Isra
 - [Getting Started](#getting-started)
 - [Supported features](#supported-features)
 - [Results](#results)
+- [Quantization Troubleshooting](#quantization-trouble-shooting)
+- [FAQ](#faq)
```

Comment:
I suggest to unify this 2 items to 1 section: Troubleshooting.
Where in this section you will refer to the 2 markdown files.